src/bo4e/com/regionalegueltigkeit.py

```diff
@@ -0,0 +1,55 @@
+"""
+Contains RegionaleGueltigkeit class
+and corresponding marshmallow schema for de-/serialization
+"""
+
+from typing import List
+
+import attr
+from marshmallow import fields, post_load
+from marshmallow_enum import EnumField  # type:ignore[import]
+
+from bo4e.com.com import COM, COMSchema
+from bo4e.com.kriteriumwert import KriteriumWert, KriteriumWertSchema
+from bo4e.enum.gueltigkeitstyp import Gueltigkeitstyp
+
+
+# pylint: disable=too-few-public-methods
+@attr.s(auto_attribs=True, kw_only=True)
+class RegionaleGueltigkeit(COM):
+    """
+    Mit dieser Komponente können regionale Gültigkeiten, z.B. für Tarife, Zu- und Abschläge und Preise definiert werden.
+    """
+
+    # required attributes
+    gueltigkeitstyp: Gueltigkeitstyp = attr.ib(
+        validator=attr.validators.in_(Gueltigkeitstyp)
+    )  #: Unterscheidung ob Positivliste oder Negativliste übertragen wird.
+    kriteriums_werte: List[KriteriumWert] = attr.ib(
+        validator=attr.validators.instance_of(List)
```

Comment:
```suggestion
        validator=attr.validators.instance_of(list)
```
ich glaube, wenn man den listeneintrag nicht genauer typisiert reicht die kleine list